package-support-policy.md

```diff
@@ -0,0 +1,90 @@
+
+### Assemblies that used to be packages, and now are part of the shared framework
+
+We have assemblies that used to be published as NuGet packages in older .NET releases, but we later made the decision of merging them into the shared framework of the next release of .NET, while at the same time we stopped publishing them as NuGet packages.
+
+If your project consumes this assembly in a version equal or greater than the one where the package was absorbed by the shared framework, then you don't need to reference the package, and you can get the latest version of the assembly whenever you upgrade your SDK or runtime to the latest version. It is not mandatory to remove the package reference from your project though, as MSBuild is capable of automatically detecting if the assembly is available in-box, in which case, the in-box version will be consumed instead, and the referenced package will be ignored.
+
+But if your project targets a .NET version that is still in support, where this assembly was still being published as a NuGet package, then as long as the package still targets an actively supported minimum .NET version, we will consider it to be actively supported until its targeted frameworks go out of support.
+
+
+### TODO: Describe the edge-case assemblies like System.Data.SqlClient, System.Xml.XPath.XDocument
+
+TODO - Assemblies that used to be In-Box and then were converted to NuGet packages (?).
+
+## Packages supported by ASP.NET Core 2.1 on .NET Framework
+
+The following document lists the minimum version you can consume of the packages that are required when using ASP.NET Core 2.1 on .NET Framework:
+
+https://dotnet.microsoft.com/en-us/platform/support/policy/aspnet/2.1-packages
+
+## Reporting issues
+
+Developers should report issues in the repo that owns the package, if they know which one it is (https://github.com/dotnet/runtime, https://github.com/dotnet/aspnetcore). When in doubt, we encourage you to report the issue directly in https://github.com/dotnet/runtime, and the repo maintainers will be happy to help transfer the issue to the rightful owners.
+
+There are some assemblies that used to be published out of a .NET version that is currently out of support, but the packages themselves are still supported because they still target a supported .NET version. For these few special cases, we host their source code in the https://github.com/dotnet/maintenance-packages repo. We still encourage users to report issues for these assemblies directly in the central repo https://github.com/dotnet/runtime.
+
+The following table lists the special case assemblies that were migrated to maintenance-packages, their branch of origin, and the way they're supported in each framework category.
+
+| Assembly | Origin | 6.0+ | .NET Standard 2.0 | 4.6.2+ |
```

Comment:
This is an interesting table for dotnet/maintenance-packages. It isn't clear to me what the support implications are.

Reply:
I see. What I want to achieve is letting users know what to expect in terms of support for packages that used to be built out of branches that are now out of support, but the packages themselves are still in support. I think the sentence in line 68 covers that, the table might not be necessary as you mention.

Reply:
I think the table is useful, but I see it more as a novelty for <1% of users.